### PR TITLE
f-footer@v4.7.0 - Updated footer component with tenant tests

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
v4.7.0
------------------------------
*January 15, 2021*

### Added
- Added new tests to test courier links are visible based on tenant.

### Changed
- Changes and refactoring to component tests that bypass the use of testing within the Storybook view.

### CODE REVIEW

File: packages/components/organisms/f-footer/test-utils/component-objects/f-footer.selctors.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const FOOTER_COMPONENT = '[data-test-id="footer-component"]'
```
Comment (simonsandersje) [inline packages/components/organisms/f-footer/test-utils/component-objects/f-footer.selctors.js]: Nit: filename selctors -> selectors

**Review by simonsandersje — DISMISSED** [review]

File: packages/components/organisms/f-footer/test/specs/component/f-footer-au.component.spec.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const locale = '&knob-Locale=en-AU'
+const links = '&knob-Show%20courier%20links=true'
+
+describe('f-footer component tests', () => {
+    beforeEach(() => {
+        browser.url(`${path}footer-component${locale}`);
+        FooterComponent.waitForFooter();
+    });
+
+    it('Does not show courier links on en-AU if courier links is set to false', () => {
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-footer/test/specs/component/f-footer-au.component.spec.js]: Start with should? `should not show courier links on en-AU if courier links is set to false` ?

File: packages/components/organisms/f-footer/test/specs/component/f-footer.component.spec.js [hunk 3]
```diff
@@ -60,11 +60,23 @@ describe('f-footer component tests', () => {
...
         browser.back();
-        browser.switchToFrame(0);
         FooterComponent.waitForFooter();
         FooterComponent.clickYoutubeIcon();
 
         // Assert
         expect(browser.getUrl()).toContain('https://www.youtube.com');
     });
+
+    it('Does not show courier links on en-GB locale if courier links is set to false', () => {
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-footer/test/specs/component/f-footer.component.spec.js]: should...

**Review by LTurns — DISMISSED** [review]

File: packages/components/organisms/f-footer/test/specs/component/f-footer.component.spec.js [hunk 4]
```diff
@@ -60,11 +60,23 @@ describe('f-footer component tests', () => {
...
+        // Assert
+        expect(FooterComponent.isCourierLinksDisplayed()).toBe(false);
+    });
+
+    it('Does not show courier links on en-GB locale if courier links is set to true', () => {
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-footer/test/specs/component/f-footer.component.spec.js]: Is this test correct? Looks like it should be asserting to `true`?

Comment (JoshuaNg2332) [reply packages/components/organisms/f-footer/test/specs/component/f-footer.component.spec.js]: Yup. This is correct. This is the GB test and we don't have any courier information to show just yet so this is supposed to assert `false`.

**Review by LTurns — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/components/organisms/f-footer/test/specs/component/f-footer-au.component.spec.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        browser.url(`${path}footer-component${locale}`);
+        FooterComponent.waitForFooter();
+    });
+
+    it('Should not show courier links on en-AU if courier links is set to false', () => {
+        // Assert
+        expect(FooterComponent.isCourierLinksDisplayed()).toBe(false);
+    });
+
+    it('Shows courier links on en-AU locale', () => {
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-footer/test/specs/component/f-footer-au.component.spec.js]: Should :)

**Review by kevinrodrigues — APPROVED** [review]

**Review by LTurns — APPROVED** [review]

**Pull request merged**